src/components/TitleTransferPanel/TitleTransferPanel.tsx

```diff
@@ -2,7 +2,11 @@ import React from "react";
 import { LabelBordered } from "../UI/LabelBordered";
 import { TitleTransferPanelContent } from "./TitleTransferPanelContent";
 
-export const TitleTransferPanel = () => {
+interface TitleTransferPanelProps {
+  document: object;
```

Comment:
Please type this.

Reply:
how do i go about doing for this case?

Reply:
There's the types in the OA package. See:

https://github.com/Open-Attestation/open-attestation/blob/master/src/%40types/document.ts
https://github.com/Open-Attestation/oa-verify/blob/master/src/verifiers/verificationBuilder.ts